source/règles/base.yaml

```
@@ -781,40 +777,61 @@
   références:
     Le salaire. Fixation et paiement: http://travail-emploi.gouv.fr/droit-du-travail/remuneration-et-participation-financiere/remuneration/article/le-salaire-fixation-et-paiement
 
-- espace: contrat salarié . salaire . brut de base
+- espace: contrat salarié . rémunération . brut de base
   nom: équivalent temps plein
-  applicable si: temps partiel
+  applicable si: temps de travail . temps partiel
   titre: Salaire brut équivalent temps plein
   résumé: Le salaire si l'embauche se faisait à temps plein
   question: Quel est le salaire en équivalent temps plein ?
   unité: €
-  formule: brut de base / quotité de travail
+  formule: brut de base / temps de travail . quotité de travail
   période: flexible
   suggestions:
     salaire médian: 2300
     SMIC: 1522
 
-- espace: contrat salarié . salaire
-  nom: brut
-  titre: Salaire brut
+- espace: contrat salarié . rémunération
+  nom: brut horaire
+  période: aucune
+  formule: rémunération . assiette de vérification du SMIC / temps de travail
```

```suggestion
  formule: rémunération . assiette de vérification du SMIC / temps de travail . temps contractuel
```

---

D'ailleurs c'est même plutôt :

```suggestion
  formule: rémunération . salaire brut / temps de travail . temps contractuel
```

non ?

---

Et du coup c'est bien `assiette de vérification du SMIC` et non `salaire brut` ? Pour moi le brut horaire correspond au salaire brut, surtout qu'il sert de base pour la rémunération des HS et qu'il est bizarre de prendre le montant des avantages en nature dans le calcul de la rémunération des HS.

---

Pardon, j'ai oublié de te répondre. D'après ce que j'ai pu lire, les avantages en nature et les primes en contrepartie directe du travail fourni sont considérés comme des éléments directe de rémunération et à ce titre, sont intégré dans le calcul du taux horaire. Il s'avère que cette distinction élément direct de rémunération / élément annexe semble être la même qui prévaut pour l'assiette de vérification du SMIC. Apparament, tout cela a été décidé par jurisprudence, d'où la difficulté de trouver des détails dessus.

---

Je rectifie, visiblement, ce n'est pas la même chose. 
Le taux horaire utilisé pour les heures supplémentaire inclue les primes de froid d'insalubrité, etc, mais pas l'assiette de vérification du SMIC.

Encore une fois, ce n'est que de la jurisprudence

---

@mquandalle j'ai fait la modification. Merci de m'avoir fait revérifier :clap: